src/main/java/org/owasp/webgoat/container/AsciiDoctorTemplateResolver.java

```diff
@@ -63,53 +68,32 @@
-        try (InputStream is = readInputStreamOrFallbackToEnglish(templateName, language)) {
-            if (is == null) {
-                log.warn("Resource name: {} not found, did you add the adoc file?", templateName);
-                return new StringTemplateResource("");
-            } else {
-                JavaExtensionRegistry extensionRegistry = asciidoctor.javaExtensionRegistry();
-                extensionRegistry.inlineMacro("webWolfLink", WebWolfMacro.class);
-                extensionRegistry.inlineMacro("webWolfRootLink", WebWolfRootMacro.class);
-                extensionRegistry.inlineMacro("webGoatVersion", WebGoatVersionMacro.class);
-                extensionRegistry.inlineMacro("webGoatTempDir", WebGoatTmpDirMacro.class);
-                extensionRegistry.inlineMacro("operatingSystem", OperatingSystemMacro.class);
-                extensionRegistry.inlineMacro("username", UsernameMacro.class);
 
-                StringWriter writer = new StringWriter();
-                asciidoctor.convert(new InputStreamReader(is), writer, createAttributes());
-                return new StringTemplateResource(writer.getBuffer().toString());
-            }
-        } catch (IOException e) {
-            //no html yet
-            return new StringTemplateResource("");
-        }
-    }
+        try (InputStream is = resourceLoader.getResource("classpath:/" + templateName).getInputStream()) {
+            JavaExtensionRegistry extensionRegistry = asciidoctor.javaExtensionRegistry();
+            extensionRegistry.inlineMacro("webWolfLink", WebWolfMacro.class);
+            extensionRegistry.inlineMacro("webWolfRootLink", WebWolfRootMacro.class);
+            extensionRegistry.inlineMacro("webGoatVersion", WebGoatVersionMacro.class);
+            extensionRegistry.inlineMacro("webGoatTempDir", WebGoatTmpDirMacro.class);
+            extensionRegistry.inlineMacro("operatingSystem", OperatingSystemMacro.class);
+            extensionRegistry.inlineMacro("username", UsernameMacro.class);
 
-    /**
-     * The resource name is for example HttpBasics_content1.adoc. This is always located in the following directory:
-     * <code>plugin/HttpBasics/lessonPlans/en/HttpBasics_content1.adoc</code>
-     */
-    private String computeResourceName(String resourceName, String language) {
-        return String.format("lessonPlans/%s/%s", language, resourceName);
-    }
-
-    private InputStream readInputStreamOrFallbackToEnglish(String resourceName, Language language) {
-        InputStream is = Thread.currentThread().getContextClassLoader().getResourceAsStream(computeResourceName(resourceName, language.getLocale().getLanguage()));
-        if (is == null) {
-            is = Thread.currentThread().getContextClassLoader().getResourceAsStream(computeResourceName(resourceName, "en"));
+            StringWriter writer = new StringWriter();
+            asciidoctor.convert(new InputStreamReader(is), writer, createAttributes());
```

Comment:
AsciiDoctor
```convert(Reader contentReader, Writer rendererWriter, Map<String, Object> options)```
is deprecated, we should use:
```convert(Reader contentReader, Writer rendererWriter, Options options)```
we should use org.asciidoctor.Options instead, maybe it's a good time for refactoring the createAttributes for using the  OptionsBuilder.